index.html

```diff
@@ -147,7 +147,7 @@ <h1>DRAFT</h1>
             </p>
         </div>
 
-        <p class="mission">The <strong>mission</strong> of the <a href="" class=todo>Linked Data Signatures Working Group</a> is to define a standard for <em><a href='https://en.wikipedia.org/wiki/Digital_signature'>digital signatures</a></em> or a unique identification of <a href="https://www.w3.org/TR/rdf11-concepts/#h2_section-dataset">RDF Datasets</a>.</p>
+        <p class="mission">The <strong>mission</strong> of the <a href="" class=todo>Linked Data Signatures Working Group</a> is to define a standard for <em><a href='https://en.wikipedia.org/wiki/Digital_signature'>digital signatures</a></em> or <em>unique identifications</em> of <a href="https://www.w3.org/TR/rdf11-concepts/#h2_section-dataset">RDF Datasets</a>.</p>
```

Comment:
```suggestion
        <p class="mission">The <strong>mission</strong> of the <a href="" class=todo>Linked Data Security Working Group</a> is to define a standard for <em><a href='https://en.wikipedia.org/wiki/Digital_signature'>digital signatures</a></em> or <em>unique identifications</em> of <a href="https://www.w3.org/TR/rdf11-concepts/#h2_section-dataset">RDF Datasets</a>.</p>
```

Hmm, perhaps we should  merge this and I can take a cut at alternative text that doesn't try to focus on digital signatures as much as this text. See comment above for background -- we don't want to accidentally give people the impression that the only thing the group is going to work on are digital signatures (e.g., canonicalization, proof of work/stake, etc.).

Reply:
And this again goes back to my reaction in #4: _this_ charter should not hint beyond signatures...